app/templates/element/header.html

```diff
@@ -0,0 +1,32 @@
+{% block header %}
+<!--Main Navigation-->
+<div id="topheader">
```

Comment:
For the header, when I run locally it displays, but is not fixed (meaning when page scrolls, header stays visible) like we discussed.

Reply:
That is not looking good, If you need i can fix that I have added that CSS in the file if you want to check, please uncomment that CSS OR i can.